app/models/defendant.rb

```diff
@@ -55,6 +55,16 @@ def defence_organisation_id
     defence_organisation&.id
   end
 
+  def judicial_result_ids
+    judicial_results.map(&:id)
+  end
+
+  def judicial_results
```

Comment:
We were considering getting defendant JRs from Hearing>defendantJudicialResults
Are you happy that we're surfacing them here instead?

Reply:
Might be worth double-checking, but I think both ways would yield the same result.

Reply:
I couldn't see a hearing that had a defendantJudicial results when I checked post writing this comment, so maybe this way is best on reflection!